src/components/map/map-layers.js

```diff
@@ -236,6 +253,8 @@ export const cartoLayers = function () {
   const basemap = createBaseMapDotAtSource('Basemap (at)', 'raster');
   // $gettext('Basemap ortho (at)', 'Map layer')
   const basemapOrtho = createBaseMapDotAtSource('Basemap ortho (at)', 'ortho');
+  // $gettext('Ordnance Survey (uk), 'Map layer')
```

Comment:
ça marche comment ces pragmas ?

Reply:
[When the](https://github.com/c2corg/c2c_ui/wiki/i18n#include-i18n-in-messages-from-api)